packages/web/.storybook/storyshots.test.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import initStoryshots, {
+  getSnapshotFileName,
+} from '@storybook/addon-storyshots';
+import renderer from 'react-test-renderer';
+import 'jest-styled-components';
```

Comment:
This library is not working as I expect. Do you know why?

Reply:
I am checking the problem XD
Btw, this plugins quite useful in my project.

Reply:
sorry, i can't find the problem why it is not working... ☹️

Reply:
I guess it is related to this. I got some warning about  multiple instances of `styled-components` when I open storybook. It's not a new issue though.
> https://github.com/styled-components/jest-styled-components#working-with-multiple-packages

Reply:
But I found that there has only one version of styled-components, are any 3rd party libraries also using the styled-components?

Reply:
Yes. Sorry I forget to mention about that.
I have included my own priority nav https://github.com/ericyip/react-priority-navigation
I tried to avoid bundle with `sc` but it's not working..

Reply:
I think you should separate them XD

Reply:
Finally got this one working
> Add custom serializer in order to work with multiSnapShot
> https://github.com/storybooks/storybook/issues/887#issuecomment-357073807